src/entity/Customer.java

```diff
@@ -2,14 +2,16 @@
 
 public class Customer {
     private int customerId;
-    private String nickName;
+    private String firstName;
+    private String lastName;
     private String address;
     private int age;
     private double discount;
 
-    public Customer(int customerId, String nickName, String address, int age, double discount) {
+    public Customer(int customerId, String firstName, String lastName, String address, int age, double discount) {
         this.customerId = customerId;
```

Comment:
Create constructor with parameters     public Customer(int customerId, String nickName, String address, int age, double discount) { and set firstName and lastName based on nickName, nickName is = "firstName.lastName"